.env.sample

```diff
@@ -15,6 +15,21 @@ JF_HOST=localhost:8080
 # JF_METRICS_IP=0.0.0.0
 # JF_METRICS_PORT=9568
 
+# Defines whether to run Jellyfish in a cluster
+# JF_DIST_ENABLED=false
+
+# Node name used in a cluster.
+# The first part can be any string.
+# The second part has to be Jellyfish address
+# it is accessible on
+# JF_NODE_NAME=jellyfish1@127.0.0.1
```

Comment:
I was wondering whether we should describe here that if you want to run all jellyfishes inside one docker network you could provide a simple service name. I think that this is worth stating because to access the container through 127.0.0.1 I think you have to export the EPMD port and multiple instances can't export it at the same time, but you could verify it.

Reply:
While I am against expanding the comments in .env files even more, I agree this should be mentioned somewhere

Reply:
I am not sure we should encourage people to do such setups :thinking:

Reply:
I agree we shouldn't encourage people to do so, but this setup is the best way to test distributed jellyfish on a single machine.

Reply:
Ok I missunderstood at first, I thought we want to run everything in one Docker.

I will describe this but I am not sure yet where